packages/pluggableWidgets/carousel-web/src/Carousel.editorConfig.ts

```diff
@@ -14,3 +27,93 @@ export function getProperties(
     }
     return defaultProperties;
 }
+
+export function getPreview(values: CarouselPreviewProps, isDarkMode: boolean): StructurePreviewProps | null {
+    const centerLayout = (props: RowLayoutProps): RowLayoutProps =>
+        ({
+            type: "RowLayout",
+            columnSize: "grow",
+            padding: 0,
+            children: [
+                {
+                    type: "Container",
+                    grow: 99,
+                    children: []
+                } as ContainerProps,
+                {
+                    type: "Container",
+                    grow: 1,
+                    children: [props]
+                },
+                {
+                    type: "Container",
+                    grow: 99,
+                    children: []
+                } as ContainerProps
+            ]
+        } as RowLayoutProps);
+    const neutralDots = [
+        {
+            type: "Image",
+            document: decodeURIComponent(DotEmpty.replace("data:image/svg+xml,", "")),
+            width: 8,
+            height: 8
+        } as ImageProps,
```

Comment:
The only thing I don't quite understand is this dot empty, why do we need it next to the dot grey? In the screenshot you send me, it also only shows 3 dots. The blue one and 2 grey ones.

Reply:
The empty dot is just to create space between dots. Because `padding` style didn't worked. So instead of adding empty container with padding (which is not working) I add empty dot (it's a same dot but transparent)